# Helper ROS message type to list available voices
# for a given text-to-speech engine.

string ttsEngine
string[] voices
